linera-views/src/backends/rocks_db.rs

```diff
@@ -53,6 +53,7 @@ struct RocksDbStoreInternal {
     root_key: Vec<u8>,
     max_stream_queries: usize,
     cache_size: usize,
+    spawn_blocking: bool,
```

Comment:
I'd rather have a boolean for the less safe optimization (that is `block_in_place: bool`) so that we can default to `false` as usual.

Reply:
Ok, renamed.

Reply:
If we expect this value to be consistent throughout the lifetime of the struct, I'd prefer if it were passed through the type system rather than be a runtime boolean.  (As an added performance bonus, it will avoid a branch every time we spawn.)

Reply:
@Twey Do you mean using an enum?
```
#[derive(Clone, Copy)]
enum RocksDbSpawnMode { SpawnBlocking, BlockInPlace }

impl RocksDbSpawnMode {
   fn spawn(self....) {
       match self { ... }
   }
}
```

Reply:
Yes, I replaced by the use of enum. However, I preferred not to use the spawn code.

Reply:
This will remove about 30 lines of code

Reply:
I did it and it worked. But it is more essential to me to get additional speed.

Using spawns (whether `span_blocking` or `block_in_place`) is forcing a clone. By focusing the clone to exactly what is needed, it seems I got some additional speed. Now we are faster at all the benchmarks (I updated in the PR description).